docker/tds/docker-compose.yml

```diff
@@ -1,5 +1,20 @@
-version:  '2'
+version:  '2.1'
```

Comment:
I generally only like to upgrade if there's something in the new version I need in order to stay as backwards compatible as possible, but I can look into the version 3 changes?